share/man/man7/c.7

```diff
@@ -55,7 +56,7 @@ different architectures.
 It follows that C is very eligible for (but not limited to) writing
 operating systems and low-level applications.
 .Pp
-The C language did not have a specification or standardized version for
+The C language did not have a specification or standardised version for
```

Comment:
This isn't related to the commit message and is an en_US -> en_GB change.

Reply:
Right, I'll split it into two commits.

Reply:
The project standard is American English https://docs.freebsd.org/en/books/fdp-primer/book/#writing-style-guidelines

Reply:
OK, one moment, please.